### DISCUSSION TITLE
Problems with exported kurviger routes in Kurviger Web

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#343] [Q&A] [answered]
Please have a look at this. I can't import route in Kurviger Format any more in Kurviger Web. Is this due to a change in the format or other problem?

https://forum.kurviger.de/t/import-error-in-kurviger-web/14360

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

Did you export the route with [Cruiser (Beta)](https://github.com/devemux86/cruiser/discussions/329)?

**Reply by DolomitesTom** [reply]
yes

**Comment by devemux86** [comment]
The weighting seems to be missing in the file.
This fixed one works: [1.3.fixed.kurviger.txt](https://github.com/devemux86/cruiser/files/12890560/1.3.fixed.kurviger.txt)

It will work properly again in the next update of Cruiser (Beta).

**Reply by DolomitesTom** [reply]
fine thank you

**Comment by devemux86** [comment]
It works properly again with the latest:
- https://github.com/devemux86/cruiser/discussions/329

**Reply by DolomitesTom** [reply]
Works now, thanks a lot for the quick fix.

**Comment by devemux86** [comment, marked as answer]
Fixed in [Cruiser 3.0.34](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
